lockfile.json

```diff
@@ -228,7 +231,7 @@
     "revision": "111b07762e86efab9a918b7c721f720c37e76b0a"
   },
   "verilog": {
-    "revision": "8f6b1f357d1231c420404b5f7a368a73c25adfa2"
+    "revision": "6fae7414fa854b5052bee9111b200e9137797f3d"
```

Comment:
please do not update the verilog parser

Reply:
https://github.com/nvim-treesitter/nvim-treesitter/runs/4335974704?check_suite_focus=true#step:10:402

Reply:
ok - i thought it is a bit harsh to do so.
I just used the ./scripts/update_lockfile.sh. I will reset all hashes but fusion

Reply:
Thank you! And please excuse that update_lockfile.sh does not the right thing automatically. On CI, we use this script with a parameter to ignore verilog.